lib/kernel/test/pg_SUITE.erl

```diff
@@ -620,13 +620,55 @@ group_leave(Config) when is_list(Config) ->
     1 = erlang:trace(PgPid, true, ['receive']),
     peer:stop(Peer),
     receive
-        {trace, PgPid, 'receive', {nodedown, Node}} -> ok
+        {trace, PgPid, 'receive', {nodedown, Node, _}} -> ok
     end,
     1 = erlang:trace(PgPid, false, ['receive']),
     sync(?FUNCTION_NAME),
     ?assertEqual([], pg:get_members(?FUNCTION_NAME, two)),
     ok.
 
+node_type(Config) when is_list(Config) ->
+    NodeTypeDefault = ?FUNCTION_NAME,
+    NodeTypeVisible = node_type_visible,
+    NodeTypeHidden = node_type_hidden,
+    NodeTypeAll = node_type_all,
+    Scopes = [NodeTypeDefault, NodeTypeVisible, NodeTypeHidden, NodeTypeAll],
+
+    {ok, _} = pg:start(NodeTypeVisible, #{node_type => visible}),
+    {ok, _} = pg:start(NodeTypeHidden, #{node_type => hidden}),
+    {ok, _} = pg:start(NodeTypeAll, #{node_type => all}),
```

Comment:
Could you please change `start` to `start_link` here?
The reason is, if the test case fails somewhere in the middle, unlinked processes will continue to run, and may cause side effects (e.g. running this test in the same shell twice fails with an unrelated reason).